Server/Application/Application.m

```diff
@@ -103,7 +103,24 @@ + (void)launchAppWithBundleId:(NSString *_Nullable)bundleId
     }
 
     application.launchArguments = launchArgs ?: @[];
-    application.launchEnvironment = environment ?: @{};
+    if ([[[UIDevice currentDevice] systemVersion] floatValue] >= 10.3) {
```

Comment:
At some point, we should extract this cod block to several methods and put it under test.